<!DOCTYPE html>
<html lang="en">
  <head>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <meta name="google-site-verification" content="f_--N3WuBtfzUzdq05UpgO98Vy6YeBD9qdJ0v6geVBg" />
	<title>Martin J. Zhang's Homepage</title>

    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <!-- Le styles -->
    <link href="assets/css/bootstrap.min.css" rel="stylesheet">
    <link href="assets/css/bootstrap-responsive.min.css" rel="stylesheet">
    <link href="assets/css/feixia.css" rel="stylesheet">

    <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
    <script src="assets/js/html5shiv.js"></script>
    <![endif]-->
  </head>

  <body>

    <div class="container">
      <div class="row">
        <div class="span3 bs-docs-sidebar">
          <!-- We use a fancy nav bar if there is enough space -->
          <hr class="hidden-phone"/>
          <ul class="nav nav-list bs-docs-sidenav hidden-phone">
            <li><a href="#"><i class="icon-play"></i>Home</a></li>     
			<li><a href="#education"><i class="icon-play"></i>Education</a></li>
			<li><a href="#publications"><i class="icon-play"></i>Publications</a></li>
			<li><a href="#softwares"><i class="icon-play"></i>Softwares </a></li>
			<li><a href="#honors"><i class="icon-play"></i>Honors</a></li>
			<!--<li><a href="#softwares"><i class="icon-play"></i>Softwares</a></li> -->
            <li><a href="assets/pdf/martin_zhang_resume.pdf"><i class="icon-play"></i>CV</a></li>
			
		  </ul>
          <hr class="hidden-phone"/>
          <div class="text-center hidden-phone">
            <img src="assets/img/mz_face.jpeg" alt="photo" class="logo-image"/>
          </div>
          <!-- Otherwise, we simply use a flat list of links -->
          <div class="visible-phone">
            <a href="#"><i class="icon-play"></i>Home</a>
			<a href="#education"><i class="icon-play"></i>Education</a>
            <a href="#publications"><i class="icon-play"></i>Publications</a>
            <a href="#honors"><i class="icon-play"></i>Honors</a>
			<!--<a href="#softwares"><i class="icon-play"></i>Softwares</a> -->
            <a href="assets/pdf/martin_zhang_resume.pdf"><i class="icon-play"></i>CV</a>
          </div>
        </div>
        <div class="span9">
          <h3>
	Martin J. Zhang
</h3>
<h5>
jinye at stanford.edu

<a href="https://github.com/martinjzhang">[Github]</a>
<a href="https://scholar.google.com/citations?user=zjr6n-QAAAAJ&hl=en">[Google Scholar]</a>

</h5>


<h3>About me</h3>
I am a fifth year PhD student at Stanford University working with Prof. <a href="http://web.stanford.edu/~dntse/">David Tse</a> and Prof. <a href="https://sites.google.com/site/jamesyzou/">James Zou</a> on computational and statistical problems that arise in biology. My background is in statistics and machine learning. My research focus on the development of statistical methods for the inference in various high-throughput genetic data like GWAS, RNA-seq, single-cell RNA-seq, and multi-omics.
I am particularly interested in methods that systematically aggregate different datasets, algorithm acceleration via adaptive sampling, and empirical Bayes modeling. 

<div>
	<b><font color="#8C1515">News:</font> </b>
	<ul>
		<li><p> 2018.12 Our <a href="https://www.biorxiv.org/content/early/2018/12/13/496372">AdaFDR</a> paper on covariate-adaptive multiple testing got accepted by RECOMB 2019 as top 5% best paper!</p></li>		
	</ul>
</div>


<h3>Education</h3>
<a name="education"></a>
<div class="media">
    <span class="pull-left"><img src="./assets/img/stanford.png" width="96px" height="96px"/></span>
    <div class="media-body">
        <p><span style="font-weight: bold">Sep. 2014 - Present</span>, Department of Electrical Engineering, <i><b>Stanford University</b></i>,</p>
        <p> PhD Candidate.</p>
    </div>
</div>

<div class="media">
    <span class="pull-left"><img src="./assets/img/tsinghua.png" width="96px" height="96px"/></span>
    <div class="media-body">
        <p><span style="font-weight: bold">Sep. 2010 - Jul. 2014</span>, Department of Electronic Engineering, <i><b>Tsinghua University</b></i>,</p>
        <p>Bachelor of Engineering.</p>
    </div>
</div>

<div class="media">
    
    <div class="media-body">
        <p><span style="font-weight: bold">Jul. 2013 - Sep. 2013</span>, Department of Electrical Engineering, <i><b>Stanford University</b></i>,</p>
        <p>Visiting Researcher. </p>
    </div>
</div>
		
		
<h3>Publications and Manuscripts</h3>
<a name="publications"></a>

<ul>
    <li><p><b>Martin J. Zhang</b>, Fei Xia, James Zou, "AdaFDR: a Fast, Powerful and Covariate-Adaptive Approach to Multiple Hypothesis Testing", accepted as top 5% best paper in <i>RECOMB 2019</i>, under review in <i>Nature Communications</i>, 2018. <a href="https://www.biorxiv.org/content/early/2018/12/13/496372">[pdf]</a> <a href="https://github.com/martinjzhang/adafdr">[software]</a> <a href="https://github.com/martinjzhang/AdaFDRpaper">[code to reproduce the paper]</a>
	</p></li> 
</ul>

<ul>
    <li><p><b>Martin J. Zhang<sup>*</sup></b>, Vasilis Ntranos<sup>*</sup>, David Tse, "One read per cell per gene is optimal for single-cell RNA-seq", under review (2nd round) in <i>Nature Communications</i>, 2018. <a href="https://www.biorxiv.org/content/early/2018/08/09/389296">[pdf]</a> <a href="https://github.com/martinjzhang/single_cell_eb">[code]</a>
	</p></li> 
</ul>

<ul>
    <li><p><b>Martin J. Zhang<sup>*</sup></b>, Abubakar Abid<sup>*</sup>, Vivek K. Bagaria, James Zou, "Exploring Patterns Unique to a Dataset with Contrastive Principal Component Analysis", <i>Nature Communications</i>, 2018. <a href="https://github.com/abidlabs/contrastive">[code]</a>
	</p></li> 
</ul>

<ul>
    <li><p>Wenyu Zhou<sup>*</sup>, M. Reza Sailani<sup>*</sup>, KÃ©vin Contrepois<sup>*</sup>, Yanjiao Zhou<sup>*</sup>, Sara Ahadi<sup>*</sup>, Shana Leopold, <b>Martin J. Zhang</b>, ..., George M. Weinstock, Michael Snyder, "Complex host-microbial dynamics in prediabetes revealed through longitudinal multi-omics profiling", under review (2nd round) in <i>Nature</i>, 2018. Contributed 4 panels in 2 figures.	</p></li> 
</ul>

<ul>
    <li><p><b>Martin J. Zhang<sup>*</sup></b>, Vivek Bagaria<sup>*</sup>, Govinda Kamath<sup>*</sup>, Vasilis Ntranos<sup>*</sup>, David Tse, "Medoids in Almost Linear Time via Multi-armed Bandits", <i>AISTATS 2018</i>. <a href="https://github.com/bagavi/Meddit">[code]</a>
	</p></li> 
</ul>

<ul>
    <li><p><b>Martin J. Zhang<sup>*</sup></b>, Fei Xia<sup>*</sup>, James Zou, David Tse, "NeuralFDR: learning decision threshold from hypothesis features",
        <i>NIPS 2017</i>. <a href="https://github.com/fxia22/NeuralFDR">[code]</a>
	</p></li> 
</ul>

<ul>
    <li><p><b>Martin J. Zhang<sup>*</sup></b>, Zhijian Ou, "Block-wise MAP Inference for the Determinantal Point Processes with Application to Change Point Detection",
        <i>SSP 2016</i>.
	</p></li> 
</ul>

<ul>
    <li><p><b>Martin J. Zhang<sup>*</sup></b>, Laming Chen, Petros T. Boufounos, and Yuantao Gu, "On the Theoretical Analysis of Cross Validation in Compressive Sensing",
        <i>ICASSP 2014</i>.
	</p></li> 
</ul>

(*Equally contributed to the project and alphabetically listed)

<h3>Softwares</h3>
<a name="softwares"></a>
<ul>
    <li><p><a href="https://github.com/martinjzhang/adafdr">adafdr</a>: covariate-adaptive multiple testing.
	</p></li> 
</ul>

<ul>
    <li><p><a href="https://github.com/martinjzhang/single_cell_eb">sceb</a>: sequencing-depth aware estimators for single-cell RNA-seq analysis via empirical Bayes.
	</p></li> 
</ul>

<ul>
    <li><p><a href="https://github.com/bagavi/Meddit">Meddit</a>: an almost linear algorithm for computing the medoid for a set of n points via adaptive sampling.
	</p></li> 
</ul>

<ul>
    <li><p><a href="https://github.com/abidlabs/contrastive">contrastive</a>: a python library for performing unsupervised machine learning on datasets with learning (e.g. PCA) in contrastive settings, where one is interested in patterns (e.g. clusters or clines) that exist one dataset, but not the other.
	</p></li> 
</ul>




<h3>Honors and Awards</h3>
<a name="honors"></a>
<ul>
<li> 2018 Top 5% best paper in RECOMB 2019 </li>
<li> 2017 NIPS travel award </li>
<li> 2015 Stanford Graduate Fellowship (SGF, Inventec Fellow) </li>
<li> 2015 Numerical Technologies Award in Electrical Engineering (Numerical Technologies Founders Graduate Fellowship) </li>
<li> 2015 Ranked 2/79 in the EE PhD Qualifying Exam at Stanford University </li>
<li> 2014 Outstanding Undergraduate Thesis "Speech Diarization Based on the Determinantal Point Processes" at Tsinghua University
<li> 2013 Comprehensive Excellence Scholarship in Electronic Engineering at Tsinghua University </li>
<li> 2011 First award in Beijing College Student Physics Competition, 3/186 in Department of Electronic Engineering at Tsinghua University </li>
<li> 2009 First Prize Provincial and Bronze Medal National, Chinese Physics Olympiad </li>
</ul>




<h3>Teaching Experiences</h3>
<a name="honors"></a>
<ul>
<li> TA, EE 278: Introduction to Statistical Signal Processing (Spring 2017)  </li>
</ul>


<!--
<h3>Press Coverage</h3>
<a href = "https://www.genomeweb.com/scan/week-genome-research-29"><img src="./assets/img/gw.png" width="80px"/> </a>
<a href = "https://medium.com/syncedreview/is-it-real-or-is-it-gibson-a-new-virtual-environment-for-ai-training-da588a707370"><img src="./assets/img/synced.png"  width="60px"/> </a>
-->

<!--
<h3>Softwares</h3>
-->

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-80823373-1', 'auto');
  ga('send', 'pageview');

</script>




        </div>
      </div>
    </div>

    <!-- Footer
    ================================================== -->
    <hr>
    <footer class="footer">
    <div class="container">
      <div class="row">
        <div class="span12">
          <p>&copy; Martin J. Zhang 2018-2020 
	     <a class="pull-right" href="#"><script type="text/javascript" id="clustrmaps" src="//cdn.clustrmaps.com/map_v2.js?d=GB91RaxcjbQAH78D8ZqC4CrkB4id3lS6DT4kESYsRTU&cl=ffffff&w=a"></script></a>
		  </p>
        </div>
      </div>
    </div>
    </footer>



    <!-- Le javascript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
<!--
    <script type="text/javascript" src="http://platform.twitter.com/widgets.js"></script>
    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
    <script src="assets/js/bootstrap-transition.js"></script>
    <script src="assets/js/bootstrap-alert.js"></script>
    <script src="assets/js/bootstrap-modal.js"></script>
    <script src="assets/js/bootstrap-dropdown.js"></script>
    <script src="assets/js/bootstrap-scrollspy.js"></script>
    <script src="assets/js/bootstrap-tab.js"></script>
    <script src="assets/js/bootstrap-tooltip.js"></script>
    <script src="assets/js/bootstrap-popover.js"></script>
    <script src="assets/js/bootstrap-button.js"></script>
    <script src="assets/js/bootstrap-collapse.js"></script>
    <script src="assets/js/bootstrap-carousel.js"></script>
    <script src="assets/js/bootstrap-typeahead.js"></script>
    <script src="assets/js/bootstrap-affix.js"></script>
    <script src="assets/js/holder/holder.js"></script>
    <script src="assets/js/application.js"></script>
-->
  </body>
</html>
